### DISCUSSION TITLE
Turbo discord appears kinda borked

### DISCUSSION THREAD

**Discussion by MrTact** [vercel/turborepo#2389] [Help] [answered]
Accepted the auto-invite but it looks like by default new users don't get any permissions to see any of the channels. It tried to throw me into the #ci-cd channel, but I get "could not load messages". (Maybe it's supposed to push new users into a welcome channel where you can accept the server agreement before you get assigned a role that can see other channels?)

**Comment by mehulkar** [comment, marked as answer]
I seem to remember there were some issues around that time, which have since been fixed. Please let us know if not!
